test.js

```diff
@@ -221,3 +221,15 @@ test('properly rounds milliseconds with secondsDecimalDigits', t => {
 	t.is(fn((3600 * 1e3) - 1), '1 hour');
 	t.is(fn((2 * 3600 * 1e3) - 1), '2 hours');
 });
+
+test('work with colon output option', t => {
+	t.is(prettyMilliseconds(1000, {colonNotation: true}), '0:01');
+	t.is(prettyMilliseconds(1500, {colonNotation: true}), '0:01.5');
```

Comment:
This does not match what the readme says, which is `5:01:45`. Why wouldn't seconds be in the same format as minutes?

Reply:
Hours, minutes and seconds are prefixed with `:`, and anything below seconds are treated with regular decimals. So `5h 1m 45s` -> ´5:01:45´, but `5h 1m 45.5s` -> `5:01:45.5` etc. See the comment below for more context.